storybook/storybook-react/src/Grid/Grid.docs.mdx

```diff
@@ -61,4 +53,47 @@ In sommige gevallen kan daar beter meer of minder witruimte staan.
 
 <Canvas of={GridStories.VerticalGap} />
 
-Voor meer voorbeelden raadpleeg je [Grid Cell](?path=/docs/react_layout-grid-cell--docs) zelf.
+### Cellen beslaan kolommen
+
+Een cel beslaat standaard 1 kolom in het grid.
+Om een cel meer kolommen te laten beslaan, gebruik je de `span` prop.
+
+<Canvas of={GridStories.CellSpan} />
+
+### Verschillende breedtes
+
+Je kunt het aantal kolommen dat een cel beslaat laten afhangen van de vensterbreedte.
+Gebruik hiervoor de `span` prop met 3 waarden voor respectievelijk smalle, middelbrede en brede vensters.
+
+Via `span={{ narrow: 4, medium: 6, wide: 8 }}` is deze cel 4 van de 4 kolommen breed op smalle vensters, 6 van de 8 op middelbrede en 8 van de 12 op brede vensters:
+
+<Canvas of={GridStories.CellSpans} />
+
+### Volledige breedte
+
+Om de cel de volledige breedte te geven – of het grid nu 4, 8, of 12 kolommen heeft – gebruik je de `fullWidth` prop.
+
+<Canvas of={GridStories.FullWidth} />
+
+### Startpositie
+
+Elke cel start automatisch in de eerstvolgende beschikbare positie in het grid.
+Een waarde van 1 hoef je dus meestal niet expliciet op te geven.
+Je kunt de startpositie van een cel aanpassen met de `start` prop.
+Zo kun je bijvoorbeeld cellen in meerdere rijen met elkaar uitlijnen.
+Of een cel in het midden van een breed grid plaatsen.
+Ook komt dit voor als een kolom als extra witruimte wilt gebruiken.
```

Comment:
Do we want to encourage this? I'd rather have people using `grid-row`

Reply:
Can you be more specific? Encourage what? Use `grid-row` how?

We offer no API for rows (yet?). Our current approach relies on automatic rows. They require less configuration to place items on the 4, 8, and 12-column grids.

Reply:
I'd rather not encourage users to use empty elements for layout purposes. We don't have an API for rows yet, but you can extend the existing css to make use of them.

Reply:
Ah, I misunderstood what this said.